torch/testing/_internal/common_quantization.py

```diff
@@ -358,6 +367,82 @@ def checkGraphModeOp(self, module, inputs, quantized_op, tracing=False, debug=Fa
 
         return models[False]
 
+    def checkGraphModuleHasNode(self, graph_module, target_node):
+        """ Check if GraphModule contains the target node
+        Args:
+            graph_module: the GraphModule instance we want to check
+            target_node: a tuple of 2 elements, first element is
+                the op type for GraphModule node, second element
+                is the target function for call_function and
+                type of the module for call_module,
+                only thse two types are supported currently
+        """
+        assert target_node[0] in ['call_function', 'call_module'], 'Only call function' \
+            ' and call module are supported right now'
+        modules = dict(graph_module.root.named_modules())
+        for node in graph_module.graph.nodes:
+            if node.op == 'call_function' and node.op == target_node[0] and node.target == target_node[1]:
+                return
+            elif node.op == 'call_module' and node.op == target_node[0] and type(modules[node.target]) == target_node[1]:
+                return
+        self.assertTrue(False, 'node:' + str(target_node) +
+                        ' not found in the graph module')
+
+    def printGraphModule(self, graph_module):
+        modules = dict(graph_module.root.named_modules())
+        for n in graph_module.graph.nodes:
+            node_info = ' '.join(map(repr, [n.op, n.name, n.target, n.args, n.kwargs]))
+            if n.op == 'call_module':
+                node_info += ' module type:' + repr(type(modules[n.target]))
+            print(node_info)
+
+    def checkGraphModeFxOp(self, model, inputs, quantized_node, quant_type=QuantType.STATIC, debug=False):
```

Comment:
optional: maybe the name could be `quantWithFxAndCheckQuantNode` or something like that?